src/Css_AtomicTypes.rei

```diff
@@ -1161,4 +1161,51 @@ module FontDisplay: {
   type t = [ | `auto | `block | `swap | `fallback | `optional];
 
   let toString: t => string;
-};
\ No newline at end of file
+};
+
+/**
+ * https://developer.mozilla.org/en-US/docs/Web/CSS/counter
+ * https://developer.mozilla.org/en-US/docs/Web/CSS/counters
+ */
+module Counter: {
+  type style = [ ListStyleType.t | `unset];
+  type t = [ | `counter(string, style) | `counters(string, string, style)];
+
+  let counter: (~style: style=?, string) => t;
+  let counters: (~style: style=?, ~separator: string=?, string) => t;
+
+  let toString: t => string;
+};
+
+/**
+ * https://developer.mozilla.org/en-US/docs/Web/CSS/counter-increment
+ * https://developer.mozilla.org/en-US/docs/Web/CSS/counter-reset
+ * https://developer.mozilla.org/en-US/docs/Web/CSS/counter-set
+ */
+module CounterOperation: {
+  type t = [ | `none | `increment(string, int) | `reset(string, int) | `set(string, int)];
```

Comment:
I would do 
```reason
module CounterIncrement : {
  type t = [ | `none | `counter(string) | `num(int) ]
  ...
```

Reply:
You mean `type t = [ | `none | `counter(string, int)  ]`? The `int` part is the number that the counter should be incrememented about.

Reply:
yes, I misread the specs.